install.rst

```diff
@@ -0,0 +1,106 @@
+Install an OMERO server
+=======================
+
+This chapter describes an installation of an OMERO.server and the most important post-installation steps, such as additional apps installation and LDAP configuration. The installation of the server on local machine is not recommended, except for testing or development purposes. The workflow is aimed at system administrators intending to familiarize themselves with OMERO installation.
+
+Description
+-----------
+
+We will show here:
+
+
+-  Where to find the OMERO.server requirements.
+
+-  How to install OMERO.server quickly using a simple Ansible playbook example.
+
+-  How to install OMERO.server very quickly using a provided Docker image.
+
+-  How to configure OMERO.server to work with LDAP, both using Ansible or using manual configuration.
+
+-  How to install apllications for OMERO.web, such as OMERO.figure.
+
+Resources
+---------
+
+Link to www OMERO at Institutions 
+Link to Simon's presentation (?)
+Link to Ansible
+Link to install docs
+Link to requrements
+Link to playbooks
+Link to App list
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
+
+Ansible
+-------
+
+For OMERO installation, we recommend to use the Ansible software suite https://docs.ansible.com/ansible/latest/index.html which enables infrastructure as a code.
+
+Install ansible (at the time of writing, the ansible-core version was 2.11.12)::
+
+    $ pip install ansible
+
+Clone the deployment examples repository::
+
+    $ git clone https://github.com/ome/omero-deployment-examples.git
+    $ cd omero-deployment-examples
+
+Clone the subrepository for the one-node example::
+
+    $ git clone https://github.com/ome/ansible-example-omero-onenode.git
+    $ cd ansible-example-omero-onenode	
+
+Install requrements::
+
+    $ ansible-galaxy install -r requirements.yml
+
+Creat a file ``hosts.yml`` in the same directory and put into it (replacing the YOUR-HOST-NAME variable as appropriate)::
```

Comment:
typo Creat

Reply:
What should `YOUR-HOST-NAME` be here? `wills_server`? Anything I like?

Reply:
Definitely not anything you like. Something like https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html (look for "hosts" in the examples). This is not trivial, I will add some formatting examples to the text as you suggest. Ultimately, it is very similar (but not the same) to "enter here your OMERO.server address" - the same level of difficulty to explain the "non-triviality". Also, having a valid host on which I can install OMERO at all is a difficult step...

Reply:
Actually, I mention in the steps above "prepare an empty box with CentOS 7 on it", I will refer to it here.